assets/lib/commands/in.rb

```diff
@@ -79,7 +79,11 @@ def pr
     end
 
     def uri
-      input.source.uri || "https://github.com/#{input.source.repo}"
+      return input.source.uri if input.source.uri
+
+      match = /https:\/\/([a-zA-Z0-9\.\-]+)/.match(input.source.api_endpoint) if input.source.api_endpoint
```

Comment:
You can specify the `uri` as `https://example.com/username/repo`, as that's what used in the `git clone` verbatim.

What problem is this solving? A test would help reflect the intention here.